### ISSUE TITLE
Hyperlink design choices are too subtle

### ISSUE THREAD

**Issue by hoj201** [broccolini/swiss#27]
I'm using this theme for my own [blog](https://www.thenewflesh.net)* and I feel like the hyperlinks blend with the text way too seamlessly in the black-theme.  The text is black on white background, and the hyperlinks are black text with a dotted underline. It would be nice if the reader had a clearer signal that there were links (e.g. by being navy blue or something).  I know it would detract from the look slightly, but it would be more functional for us bloggers.

*Credit yourself in the footer.  This theme is beautiful, and I'm sure some people will want to see who designed it.

**Comment by TheVictorMunoz** [comment]
I agree with this comment and actually I would like to know how to edit it?

Also, I would like to know how to change the size of the title to make it a bit smaller? 

Thanks in advance!

**Comment by ntomita** [comment]
@hoj201  Regarding the hyperlink color, you should modify parameters for a tag which you can find in _sass/_base.scss:
```
a {
  color: $color-primary-link;
  text-decoration: none;
}
```
@vmmunoza I agree that the default title font size is unreasonably huge. You can set like
`$h0-desktop: 72px;
`
in _sass/_variables.scss.

**Comment by Ralph-Lee** [comment]
@vmmunoza Finally, I found the best solution here [jekyllrb.com](https://jekyllrb.com/docs/themes/#overriding-theme-defaults):

1. `cd /path/to/your/repo/_layouts/`
2. `cp $(bundle info swiss --path)/_layouts/post.html ./`
3. `cp $(bundle info swiss --path)/_layouts/category-post.html`
4. change line **5** to `<h1 class="col-9 sm-width-full py-4 mt-3 inline-block" itemprop="name headline">{{ page.title }}</h1>`
5. run `bundle exec jekyll serve`
